squirrel.api/src/main/java/org/dice_research/squirrel/queue/UriQueue.java

```diff
@@ -47,4 +47,9 @@
      * Opens the queue and allocates necessary resources.
      */
     public void open();
+    
+    /**
+     * Check if the queue is storing the depth
+     */
+    public boolean isDepthIncluded();

```

Comment:
I don't get why this method is necessary

Reply:
this is was for an implementation that i had in mind, but forgot to remove